### PR TITLE
Kata refactoring

### PR DESCRIPTION
**Pull request by valtair1891**

### CODE REVIEW

File: testdata/kata/release-4.9/example.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    app: httpd
+  name: example
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  containers:
+    -
+      image: "image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest"
+      name: httpd
+      ports:
+	-
```
Comment (pruan-rht) [inline testdata/kata/release-4.9/example.yaml:14]: @valtair1891 what is this special character for?

**Review by pruan-rht — COMMENTED** [review]
@valtair1891 ` features/node/kata/pod.feature → features/kata/kata_features/pod.feature` IMO, I don't think you need the `kata_features` directory since `feature/kata/` is sufficient.

**Comment by pruan-rht** [comment]
@valtair1891 github best practice is to squash your commits to as few commits as possible before submitting them for review.  https://cloudfour.com/thinks/squashing-your-pull-requests/

**Comment by valtair1891** [comment]
Breaking down to a smaller commits

**Pull request closed without merging**